packaging/ubuntu-16.04/control

```diff
@@ -20,9 +20,11 @@ Build-Depends: autoconf,
                golang-any (>=2:1.6) | golang-1.6,
                indent,
                init-system-helpers,
-               libcap-dev,
                libapparmor-dev,
+               libcap-dev,
+               libfuse-dev,
```

Comment:
I see this is in 16.04 build, what about 14.04? I know it doesn't build but do you anticipate any problems due to its absence?